src/applications/edu-benefits/10203/content/SaveInProgressIntro.jsx

```diff
@@ -0,0 +1,381 @@
+import React from 'react';
```

Comment:
what are the major differences between this and `platform/forms/save-in-progress/SaveInProgressIntro` ?

Reply:
Color of the button and some text change